src/js/containers/agencyV2/visualizations/ObligationsByAwardTypeContainer.jsx

```diff
@@ -59,12 +59,12 @@ export default function ObligationsByAwardTypeContainer({ fiscalYear, windowWidt
                 dispatch(setAwardObligations(res.data.total_aggregated_amount));
                 const categories = [
                     {
-                        label: ['Financial', 'Assistance'], // line break between words
+                        label: ['All Financial', 'Assistance'], // line break between words
```

Comment:
I guess `All` is coming from a design review since the mock do not have that?

Reply:
correct